src/pollution_model/run_medusa_2.py

```diff
@@ -426,8 +433,55 @@ def store_pollution_model_in_database(engine: Engine, results: gpd.GeoDataFrame,
         The id of the current medusa2.0 model run, to associate with the results.
     """
     results["scenario_id"] = scenario_id
-    results.set_index("Index", inplace=True)
-    results.to_sql("medusa2_model_output", engine, if_exists="append", index=True)
+    results.index.names = ["spatial_feature_id"]
+    results.to_sql(MEDUSA2ModelOutput.__tablename__, engine, if_exists="append", index=True)
+
+
+def serve_pollution_model() -> None:
+    """Serve the geospatial data for pollution models visualisation and API use.
+    Joins the medusa2_model_output table to the corresponding geospatial feature tables.
+    """
+    # Create geoserver workspace for pollution data
+    db_name = EnvVariable.POSTGRES_DB
+    workspace_name = f"{db_name}-pollution"
+    geoserver.create_workspace_if_not_exists(workspace_name)
+
+    # Ensure workspace has access to database
+    data_store_name = f"{db_name} PostGIS"
+    geoserver.create_db_store_if_not_exists(db_name, workspace_name, data_store_name)
```

Comment:
It would be beneficial to update the logging in the following GeoServer functions for consistency: `create_workspace_if_not_exists`, `create_db_store_if_not_exists`, and `create_datastore_layer`. Currently, the logs vary based on whether a workspace or store exists:

When neither the workspace nor store exists, the logs are:
```python
2024-08-14 11:06:07 | INFO     | src.geoserver.geoserver_common   47 | create_workspace_if_not_exists                     | Creating geoserver workspace db-pollution if it does not already exist.
2024-08-14 11:06:08 | INFO     | src.geoserver.geoserver_common   59 | create_workspace_if_not_exists                     | Created new workspace db-pollution.
2024-08-14 11:06:19 | INFO     | src.geoserver.database_layers   155 | create_db_store_if_not_exists                      | Created new db store db-pollution.
2024-08-14 11:07:18 | INFO     | src.geoserver.database_layers    89 | create_datastore_layer                             | Created new datastore layer db-pollution:medusa2_model_output.
```

When the workspace or store already exists, the logs are:
```python
2024-08-14 11:08:24 | INFO     | src.geoserver.geoserver_common   47 | create_workspace_if_not_exists                     | Creating geoserver workspace db-pollution if it does not already exist.
2024-08-14 11:08:24 | DEBUG    | src.geoserver.geoserver_common   61 | create_workspace_if_not_exists                     | Workspace db-pollution already exists.
```

Reply:
How does this look?

```python
INFO     | src.geoserver.geoserver_common   47 | create_workspace_if_not_exists                     | Creating geoserver workspace "db-pollution" if it does not already exist.
INFO     | src.geoserver.geoserver_common   59 | create_workspace_if_not_exists                     | Created new workspace "db-pollution".
INFO     | src.geoserver.database_layers   155 | create_db_store_if_not_exists                      | Creating database store "db-pollution:db PostGIS" if it does not already exist.
INFO     | src.geoserver.database_layers   155 | create_db_store_if_not_exists                      | Created new database store "db-pollution:db PostGIS".
INFO     | src.geoserver.database_layers    89 | create_datastore_layer                             | Creating new datastore layer "db-pollution:medusa2_model_output" if it does not exist.
INFO     | src.geoserver.database_layers    89 | create_datastore_layer                             | Created new datastore layer "db-pollution:medusa2_model_output".
```

and

```python
INFO     | src.geoserver.geoserver_common   47 | create_workspace_if_not_exists                     | Creating geoserver workspace "db-pollution" if it does not already exist.
DEBUG    | src.geoserver.geoserver_common   61 | create_workspace_if_not_exists                     | Workspace "db-pollution" already exists.
INFO     | src.geoserver.database_layers   155 | create_db_store_if_not_exists                      | Creating database store "db-pollution:db PostGIS" if it does not already exist.
DEBUG    | src.geoserver.database_layers   155 | create_db_store_if_not_exists                      | Database store "db-pollution:db PostGIS" already exists.
INFO     | src.geoserver.database_layers    89 | create_datastore_layer                             | Creating new datastore layer db-pollution:medusa2_model_output if it does not exist.
DEBUG     | src.geoserver.database_layers    89 | create_datastore_layer                             | Datastore layer "db-pollution:medusa2_model_output" already exists.
```

Reply:
Looks good :)